catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/converter/MysqlTypeConverter.java

```diff
@@ -7,16 +7,14 @@
 import com.datastrato.gravitino.catalog.jdbc.converter.JdbcTypeConverter;
 import com.datastrato.gravitino.rel.types.Type;
 import com.datastrato.gravitino.rel.types.Types;
-import java.util.List;
-import net.sf.jsqlparser.statement.create.table.ColDataType;
 
 /** Type converter for MySQL. */
-public class MysqlTypeConverter extends JdbcTypeConverter<ColDataType, String> {
+public class MysqlTypeConverter
+    extends JdbcTypeConverter<MysqlTypeConverter.MysqlTypeBean, String> {
```

Comment:
Why is MysqlTypeBean necessary? Can we convert to Gravitino type from string or column information directly?